cellpack/autopack/ingredient/single_cube.py

```diff
@@ -12,118 +12,95 @@
             cutoff_surface=cutoff_surface,
-            distExpression=distExpression,
-            distFunction=distFunction,
-            excluded_partners_name=excluded_partners_name,
+            distance_expression=distance_expression,
+            distance_function=distance_function,
             force_random=force_random,
             gradient=gradient,
-            isAttractor=isAttractor,
-            jitterMax=jitterMax,
+            is_attractor=is_attractor,
+            max_jitter=max_jitter,
             molarity=molarity,
             name=name,
-            nbJitter=nbJitter,
-            nbMol=nbMol,
-            overwrite_distFunc=overwrite_distFunc,
-            packingMode=packingMode,
-            packingPriority=packingPriority,
-            partners_name=partners_name,
-            partners_position=partners_position,
-            pdb=pdb,
-            perturbAxisAmplitude=perturbAxisAmplitude,
-            placeType=placeType,
-            positions2=positions2,
-            positions=positions,
-            principalVector=principalVector,
-            proba_binding=proba_binding,
-            proba_not_binding=proba_not_binding,
-            properties=properties,
-            radii=radii,
-            rotAxis=rotAxis,
-            rotRange=rotRange,
-            useRotAxis=useRotAxis,
+            jitter_attempts=jitter_attempts,
+            overwrite_distance_function=overwrite_distance_function,
+            packing_priority=packing_priority,
+            partners=partners,
+            perturb_axis_amplitude=perturb_axis_amplitude,
+            place_type=place_type,
+            principal_vector=principal_vector,
+            representations=representations,
+            rotation_axis=rotation_axis,
+            rotation_range=rotation_range,
+            use_rotation_axis=use_rotation_axis,
             weight=weight,
         )
 
         if name is None:
-            name = "%5.2f_%f" % (radii[0][0], molarity)
+            name = "%5.2f_%f" % (bounds[0][0], molarity)
         self.name = name
-        self.singleSphere = False
-        self.modelType = "Cube"
+        self.model_type = "Cube"
         self.collisionLevel = 0
-        radii = numpy.array(radii)
 
-        self.minRadius = min(radii[0] / 2)  # should have three radii sizex,sizey,sizez
-        self.maxRadius = self.encapsulatingRadius = numpy.linalg.norm(
-            radii[0] / 2
-        )  # calculate encapsulating radius based on side length
-        self.bb = [-radii[0] / 2, radii[0] / 2]
-
-        self.positions = positions  # bottom left corner of cuboid
-        self.positions2 = positions2  # top right corner of cuboid
-        positions_ar = numpy.array(self.positions[0][0])
-        positions2_ar = numpy.array(self.positions2[0][0])
+        self.bb = bounds
 
+        self.lower_bound = bounds[0]  # bottom left corner of cuboid
+        self.upper_bound = bounds[1]  # top right corner of cuboid
```

Comment:
I think this leaves open the ability for `self.bb` to diverge/differ from `self.lower_bound/self.upper_bound`? Could be good to merge these into a single source of truth at some point.